models/migrations/v160.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
```

Comment:
I've updated the license header for this file, but could you add this migration to the v1_19 sub-directory?

Reply:
Moved the v160.go into v1_19 sub-directory and renamed to v236.go (hope it's ok) tell me if v999.go or some other name is more suitable for now